azure-pipelines.yml

```diff
@@ -160,7 +166,7 @@ stages:
         displayName: Push tools to feed
         inputs:
           command: 'push'
-          packagesToPush: '$(Pipeline.Workspace)/{tools}/*.nupkg'
+          packagesToPush: '$(Pipeline.Workspace)/tools/*.nupkg'
```

Comment:
This is the main fix. Unfortunately, it only gets hit in main, so didn't catch it in the PR. The brackets here and the next change are probably all that is required.